include/solvers/fluid_dynamics_matrix_free_operators.h

```diff
@@ -785,4 +825,51 @@ class NavierStokesStabilizedOperator
     const std::pair<unsigned int, unsigned int> &range) const override;
 };
 
+/**
+ * @brief Implements the matrix-free operator to solve steady/transient Non-newtonian Navier-Stokes equations using stabilization.
```

Comment:
We could add a small disclaimer saying we only have SUPG-PSPG for the time being